### DISCUSSION TITLE
Build on Fedora Silverblue or Fedora kinoite

### DISCUSSION THREAD

**Discussion by moacirrf** [berarma/new-lg4ff#95] [Show and tell]
Hi 
I'm using Fedora Kinoite that is an immutable distribution, so i created a script to help build and install this module.

https://github.com/moacirrf/new-lg4ff-fedora-silverblue

Hope this help someone.

Thanks @berarma  for this great job

**Comment by mwprado** [comment]
> Hi 
> I'm using Fedora Kinoite that is an immutable distribution, so i created a script to help build and install this module.
> 
> https://github.com/moacirrf/new-lg4ff-fedora-silverblue
> 
> Hope this help someone.
> 
> Thanks @berarma  for this great job

Is there a possibility to submit to bazzite repository?

**Reply by moacirrf** [reply]
humm Interesting project...
